docs/user-guide/vwii/sd-preparation.md

```diff
@@ -10,22 +10,23 @@ We will now start by placing the required Homebrew files on the SD Card.
     Your SD Card will need to be formatted as FAT32. If your SD Card is not formatted to FAT32, use [GUIFormat](http://ridgecrop.co.uk/index.htm?guiformat.htm) with 32k (32768) Allocation unit size to format it. **Do not** label the SD Card as `wiiu` or it will cause issues with homebrew.
     
 ?> If you have hacked your Wii U in the past, you can use the same SD Card for this process.
-    
+
+?> Unlike Wii U homebrew, vWii apps go in a separate apps folder on the root. Compat Installer is the **only** Wii U app here. 
 
 ### What You Need
 
-- The latest files from [Tiramisu for your café](https://tiramisu.foryour.cafe).
-    - Click on `Download Tiramisu`.
-    - If you have already installed Tiramisu, you do not need to redownload it.
+- The latest files from [Aroma for your café](https://aroma.foryour.cafe).
```

Comment:
Should probably have a note for the checkboxes as well.

Reply:
Would it be better as a bullet point or an info box

Reply:
bullet, like the main guide.

Reply:
Sounds good